webpack.config.js

```diff
@@ -1,5 +1,6 @@
 /* eslint-env node */
 /* eslint-disable import/no-commonjs */
+/* eslint-disable import/order */
```

Comment:
eslint really wanted to reorder some imports here in ways that I'm not sure is safe.